Leanplum-SDK/Classes/LPRequestStorage.m

```diff
@@ -97,8 +97,8 @@ - (void)migrateRequests
  */
```

Comment:
RequestStorage uses plist (XML). We are deprecating this. Let me know if you want something specific.